src/pages/home/report/ReportActionItemMessage.js

```diff
@@ -18,15 +18,25 @@ const propTypes = {
     /** Information about the network */
     network: networkPropTypes.isRequired,
 
+    /** Additional styles to add after local styles. */
+    style: PropTypes.oneOfType([
+        PropTypes.arrayOf(PropTypes.object),
+        PropTypes.object,
+    ]),
+
     /** localization props */
     ...withLocalizePropTypes,
 };
 
+const defaultProps = {
+    style: [],
+};
+
 const ReportActionItemMessage = (props) => {
     const isUnsent = props.network.isOffline && props.action.isLoading;
 
     return (
-        <View style={[styles.chatItemMessage, isUnsent && styles.chatItemUnsentMessage]}>
+        <View style={[styles.chatItemMessage, isUnsent && styles.chatItemUnsentMessage, props.style]}>
```

Comment:
usually seeing arrays get spread into the one array - but I guess maybe the styles are flattened somewhere. Maybe for consistency we change to this:

```suggestion
        <View style={[styles.chatItemMessage, isUnsent && styles.chatItemUnsentMessage, ...props.style]}>
```